### ISSUE TITLE
[Bug][UI] Monitor gauge chart  display blank  with same pid

### ISSUE THREAD

**Issue by geosmart** [apache/incubator-dolphinscheduler#3548]
**Describe the bug**
with correct response data , the monitor chart doesn't show as expected.

**Expected behavior**
just show the correct chart rather than blank

the two worker start by docker ,and have the same pid.
worker in different machine has same pid is ok, so the ui show blank is a bug

**Screenshots**
![chart](https://user-images.githubusercontent.com/3156608/90588557-7b0b9e80-e20e-11ea-9df1-641e02ee6298.png)

![response json](https://user-images.githubusercontent.com/3156608/90588551-7810ae00-e20e-11ea-968d-835d1ba3f641.png)

**Which version of Dolphin Scheduler:**
 -[1.3.2-preview]

**Comment by yangyichao-mango** [comment]
Hi～
Please try clear the site cache and try it again.Multi version ui maybe have the left cache in your browser.

**Comment by geosmart** [comment]
```json
{
    "code": 0,
    "msg": "成功",
    "data": [
        {
            "id": 31,
            "host": "10.199.150.91",
            "port": 5678,
            "zkDirectory": "/dboard/nodes/master/10.199.150.91:5678",
            "resInfo": "{\"cpuUsage\":0.02,\"loadAverage\":0.19,\"memoryUsage\":0.41}",
            "createTime": "2020-08-20T07:05:17.000+0000",
            "lastHeartbeatTime": "2020-08-20T07:31:53.000+0000"
        },
        {
            "id": 30,
            "host": "172.24.0.4",
            "port": 5678,
            "zkDirectory": "/dboard/nodes/master/172.24.0.4:5678",
            "resInfo": "{\"cpuUsage\":0.03,\"loadAverage\":0.01,\"memoryUsage\":0.25}",
            "createTime": "2020-08-20T06:42:30.000+0000",
            "lastHeartbeatTime": "2020-08-20T07:31:54.000+0000"
        },
        {
            "id": 30,
            "host": "10.199.150.90",
            "port": 5678,
            "zkDirectory": "/dboard/nodes/master/10.199.150.90:5678",
            "resInfo": "{\"cpuUsage\":0.0,\"loadAverage\":0.14,\"memoryUsage\":0.48}",
            "createTime": "2020-08-20T07:05:21.000+0000",
            "lastHeartbeatTime": "2020-08-20T07:31:47.000+0000"
        }
    ]
}
```
clear the cache, use new browser,still not show as expect.
the last one show blank ,may be the cpuUsage=0 make something wrong?

**Comment by geosmart** [comment]
finaly I found the key, the two worker start by docker ,and have the same pid.
worker in different machine has same pid is ok, so this is a bug

**Comment by geosmart** [comment]
> finaly I found the key, the two worker start by docker ,and have the same pid.
> worker in different machine has same pid is ok, so this is a bug

**Comment by geosmart** [comment]
fixed [#3563 ](https://github.com/apache/incubator-dolphinscheduler/pull/3563/commits/303a8d4aa1162fc09f35bf7dde3bd1e7e68e448d)

**Comment by geosmart** [comment]
fixed [#3563 ](https://github.com/apache/incubator-dolphinscheduler/pull/3563/commits/303a8d4aa1162fc09f35bf7dde3bd1e7e68e448d)
